src/audio/PlaylistManager.js

```diff
@@ -273,6 +289,9 @@ FORGE.PlaylistManager.prototype._applyConfig = function(config)
 
         this._maxVolume = (typeof config.volume !== "undefined" && typeof config.volume.max === "number") ? FORGE.Math.clamp(config.volume.max, 0, 1) : 1;
         this._volume = (typeof config.volume !== "undefined" && typeof config.volume.default === "number") ? FORGE.Math.clamp(config.volume.default, 0, this._maxVolume) : FORGE.Math.clamp(1, 0, this._maxVolume);
+
+        this._autoPause = (typeof config.autoPause !== "undefined") ? !!config.autoPause : null;
+        this._autoResume = (typeof config.autoResume !== "undefined") ? !!config.autoResume : null;
```

Comment:
Same as before :)
this._autoPause = (typeof config.autoPause == "boolean") ? config.autoPause : defaultValue;